tests/lib/bash/test_header

```diff
@@ -495,7 +495,7 @@ mock_smtpd_init() {  # Logic borrowed from Rose
     for SMTPD_PORT in 8025 8125 8225 8325 8425 8525 8625 8725 8825 8925; do 
         local SMTPD_HOST="localhost:${SMTPD_PORT}"
         local SMTPD_LOG="${TEST_DIR}/smtpd.log"
-        python3 -m 'smtpd' -c 'DebuggingServer' -d -n "${SMTPD_HOST}" \
+        python3 -u -m 'smtpd' -c 'DebuggingServer' -d -n "${SMTPD_HOST}" \
```

Comment:
I don't know if this would work, could we `export PYTHONUNBUFFERED` somewhere at the top level of the project to avoid needing to do this?

Reply:
Why wouldn't it work?  `man python3` says:
```
    PYTHONUNBUFFERED
              If this is set to a non-empty string it is equivalent to specifying the -u option.
```

Reply:
I guess I was think setting this specifically where needed is better than generically (although it doesn't really matter ... I doubt we're relying on output buffering anyway in the tests!)

Reply:
Let me know if you disagree.

Reply:
> setting this specifically where needed is better than generically 

OK, if its not desired globally that's fine.